src/usexp/template.mli

```diff
@@ -0,0 +1,27 @@
+type var_syntax = Usexp0.Template.var_syntax = Dollar_brace | Dollar_paren | Percent
+
+type var = Usexp0.Template.var =
+  { loc: Loc.t
+  ; name: string
+  ; payload: string
```

Comment:
This should be a `string option`

Reply:
It was actually an option previously, but then you suggested the set of types that assigned this field a `string`.

Reply:
Ah, that was a mistake. I probably didn't realize it at the time as I didn't try to use these types